File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i=0,j=numbers.length-1;
        while(i<j){
            if(target-numbers[i]-numbers[j]==0){
                return new int[]{i+1,j+1};
            }
            else if(target-numbers[i]-numbers[j]>0){
                i++;
            }
            else{j--;}
        }
        return null;
    }
}

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        n= len(numbers)
        L=0
        R=n-1

        while L<R:
            if (numbers[L]+numbers[R]==target):
                return [L+1, R+1]
            elif(numbers[L]+numbers[R]>target):
                R-=1
            elif (numbers[L]+numbers[R]<target):
                L+=1
            